src/main/java/com/kainos/ea/controller/TrainingController.java

```diff
@@ -0,0 +1,48 @@
+package com.kainos.ea.controller;
+
+import com.kainos.ea.dao.TrainingDao;
+import com.kainos.ea.exception.DatabaseConnectionException;
+import com.kainos.ea.exception.FailedToGetTrainingException;
+import com.kainos.ea.exception.TrainingDoesNotExistException;
+import com.kainos.ea.service.TrainingService;
+import com.kainos.ea.util.DatabaseConnector;
+import io.swagger.annotations.Api;
+import org.eclipse.jetty.http.HttpStatus;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.sql.SQLException;
+
+@Api("Backend API Training")
+@Path("/api")
+public class TrainingController {
+
+    private TrainingService trainingService;
+
+    public TrainingController() {
+        trainingService = new TrainingService(new TrainingDao(), new DatabaseConnector());
+    }
+
+    public TrainingController(TrainingService trainingService) {
+        this.trainingService = trainingService;
+    }
+
+    @GET
+    @Path("/training/{id}")
```

Comment:
could we rename this to be bandId?
```suggestion
    @Path("/training/{bandId}")
```